packages/replay-next/components/inspector/values/ErrorRenderer.tsx

```diff
@@ -18,5 +57,6 @@ export function errorProtocolObjectToString(protocolObject: ProtocolObject) {
     property => property.name === "message"
   );
 
+  // TODO We'd need to suspend / await to find the getter value. Skip that case for now.
```

Comment:
Could you add more detail to this comment? Like a link the Linear issue at least? It's kind of vague.

Reply:
There's no separate issue.  This is just me noting that the logic you wrote for doing the "copy to string" functionality _appears_ to be synchronous at first glance, so I didn't want to follow that rabbit hole and try to do something async here.

Reply:
Okay fair. I was just saying that this comment could use more detail. It's pretty vague.

Reply:
Double-checked how the "copy to string" logic works, and I see that the parent function _is_ `async` already.  So, I went ahead and converted this function to be `async` and do the same "fetch the getter" logic as the component does.

Reply:
Nice 👍🏼